### DISCUSSION TITLE
Admin UI RTL

### DISCUSSION THREAD

**Discussion by yalondpsi** [keystonejs/keystone#7716] [Questions] [answered]
How can I "inject" to the Admin UI body tag - `dir="RTL" `?
(I need the Admin UI to be RTL aligned)
Thanks

**Comment by yalondpsi** [comment, marked as answer]
I succeded 
If anyone wants a detailed answer - he can approach me
Thanks

**Reply by Thebinoman** [reply]
I also have a case with rtl language. A detailed answer would be very helpful. Thank you!
